app/display.c

```diff
@@ -69,14 +73,16 @@ void display_init(void)
 
     clear_screen();
 
-    set_foreground_colour(RED);
-    set_background_colour(WHITE);
+    set_foreground_colour(BLACK);
+    set_background_colour(GREEN);
```

Comment:
This is unrelated to serial/TTY support, so belongs in a separate commit.

Reply:
Correct. That belong to the aesthetic branch (and also the red + some line below). Anyway, I'll try to push that quickly to main because the actual color confuses some betatesters (I still get bug reports for passmark's memtest86 :-/). Removing here.

Reply:
I find black on green hard to read which is why I changed the colour scheme.

Reply:
I also have the same concern about the RED/WHITE color scheme :-/ 

Anyway, I don't have any fetish for the green color. It was useful for a very long time because the original memtest86 was red and memtest86+ was green, so anyone could tell at a glance (even from a distance) if a system was running memtest86 or memtest86+. 

But now, passmark changed the title for their memtest86 from red to green so that didn't work anymore.

I'll do some tries next week and ask for feedbacks